ipaserver/rpcserver.py

```diff
@@ -493,7 +493,7 @@ def marshal(self, result, error, _id=None,
 
     def unmarshal(self, data):
         try:
-            d = json.loads(data)
+            d = json.loads(ipautil.decode_json(data))
```

Comment:
@tiran do you agree to hardcode everything into 'utf-8' for now? (we can open a ticket to support more encodings)